cfgov/mmt_cost_of_credit_tool/views.py

```diff
@@ -0,0 +1,21 @@
+from django.views.generic.base import TemplateView
```

Comment:
This class is a duplicate of one that already exists in the My Money Calendar tool's Python module path. You can `git rm` this and it'll still work correctly. 

We may want to move the original one into a more central location and change its import statement in urls.py since both tools might use it, but that can happen later. For now, just delete this file from revision control.

Reply:
I didn't edit this.  Are you sure?